frameworks/Java/tapestry/install.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-fw_depends java7 resin maven
+fw_depends java8 resin-java8 maven
```

Comment:
We discussed this offline and apparently resin needs to be compiled with the same java runtime that is used to invoke it. So, this `resin-java8` must exist unless we move EVERYTHING to java8 explicitly. Very annoying.